recipes/soxr/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "soxr" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/{{ name }}/{{ name }}-{{ version }}-Source.tar.xz
+  sha256: b111c15fdc8c029989330ff559184198c161100a59312f5dc19ddeb9b5a15889
+
+build:
+  number: 0
+  ignore_run_exports:
+    - libstdcxx-ng
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
```

Comment:
```suggestion
```

As suggested by @isuruf

Reply:
done in https://github.com/conda-forge/staged-recipes/pull/17935/commits/e315dc9fce53e45c4c4113d4c9baea7f94cb6fb7